<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="NPAND1W3" name="OCTS_Automation_Main_Modules.ERP_Financial_Webservice_MainClass" tests="1" failures="0" timestamp="22 Nov 2017 23:40:15 GMT" time="9.299" errors="1">
  <testcase name="OracleCloud_Financial_Webservices" time="9.299" classname="OCTS_Automation_Main_Modules.ERP_Financial_Webservice_MainClass">
    <error type="java.lang.Exception" message="UploadFileToUcm: Error occurred while sending SOAP Request to Server">
      <![CDATA[java.lang.Exception: UploadFileToUcm: Error occurred while sending SOAP Request to Server
	at OCTS_Automation_Main_Modules.ERP_Financial_Webservice_MainClass.OracleCloud_Financial_Webservices(ERP_Financial_Webservice_MainClass.java:301)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at OCTS_Automation_Main_Modules.TestNG_Invoke_WS.wstriggertestng(TestNG_Invoke_WS.java:92)
	at OCTS_GUI.FXMLController$1.call(FXMLController.java:215)
	at javafx.concurrent.Task$TaskCallable.call(Task.java:1423)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- OracleCloud_Financial_Webservices -->
</testsuite> <!-- OCTS_Automation_Main_Modules.ERP_Financial_Webservice_MainClass -->
